phpBB/styles/prosilver/template/overall_header.html

```diff
@@ -84,7 +84,9 @@
 			<div class="inner">
 
 			<div id="site-description" class="site-description">
-				<a id="logo" class="logo" href="<!-- IF U_SITE_HOME -->{U_SITE_HOME}<!-- ELSE -->{U_INDEX}<!-- ENDIF -->" title="<!-- IF U_SITE_HOME -->{L_SITE_HOME}<!-- ELSE -->{L_INDEX}<!-- ENDIF -->"><span class="site_logo"></span></a>
+				<a id="logo" class="logo" href="{% if U_SITE_HOME %}{{ U_SITE_HOME }}{% else %}{{ U_INDEX }}{% endif %}" title="{% if U_SITE_HOME %}{{ lang(L_SITE_HOME) }}{% else %}{{ lang(L_INDEX) }}{% endif %}">
```

Comment:
`{{ lang(L_SITE_HOME) }}` and `{{ lang(L_INDEX) }}`
should be
`{{ lang('SITE_HOME') }}` and `{{ lang('INDEX') }}`